docs/how-to/internal/release.rst

```diff
@@ -43,12 +43,13 @@ The procedure for cutting a new release is as follows:
    * briefcase-linux-flatpak-template
    * briefcase-iOS-Xcode-template
    * briefcase-android-gradle-template
+   * briefcase-web-static-template
 
-4. Tag a release candidate, and push the branch and tag upstream::
+4. Tag the release, and push the branch and tag upstream::
```

Comment:
Since all packages now go through the full automated test process, the chance of anything going wrong during manual testing is much smaller, and there's no need to complicate the release process by making an RC build first.